Examples/Tests/scraper/PICMI_inputs.py

```diff
@@ -0,0 +1,209 @@
+    scattering_processes={
+        'elastic' : {
+            'cross_section' : cross_sec_direc+'electron_scattering.dat'
+        },
+        'excitation1' : {
+            'cross_section': cross_sec_direc+'excitation_1.dat',
+            'energy' : 19.82
+        },
+        'excitation2' : {
+            'cross_section': cross_sec_direc+'excitation_2.dat',
+            'energy' : 20.61
+        },
+        'ionization' : {
+            'cross_section' : cross_sec_direc+'ionization.dat',
+            'energy' : 24.55,
+            'species' : ions
+        },
+    }
+)
+
+mcc_ions = picmi.MCCCollisions(
+    name='coll_ion',
+    species=ions,
+    background_density=N_INERT,
+    background_temperature=T_INERT,
+    scattering_processes={
+        'elastic' : {
+            'cross_section' : cross_sec_direc+'ion_scattering.dat'
+        },
+        'back' : {
+            'cross_section' : cross_sec_direc+'ion_back_scatter.dat'
+        },
+        # 'charge_exchange' : {
+        #    'cross_section' : cross_sec_direc+'charge_exchange.dat'
+        # }
+    }
+)
+
+##########################
+# numerics components
+##########################
+
+grid = picmi.Cartesian2DGrid(
+    number_of_cells = [nx, ny],
+    lower_bound = [xmin, ymin],
+    upper_bound = [xmax, ymax],
+    bc_xmin = 'dirichlet',
+    bc_xmax = 'dirichlet',
+    bc_ymin = 'periodic',
+    bc_ymax = 'periodic',
+    warpx_potential_hi_x = "%.1f*sin(2*pi*%.5e*t)" % (VOLTAGE, FREQ),
+    lower_boundary_conditions_particles=['absorbing', 'periodic'],
+    upper_boundary_conditions_particles=['absorbing', 'periodic'],
+    moving_window_velocity = None,
+    warpx_max_grid_size = nx//4
+)
+
+solver = picmi.ElectrostaticSolver(
+    grid=grid, method='Multigrid', required_precision=1e-6
+)
+
+##########################
+# diagnostics
+##########################
+
+field_diag = picmi.FieldDiagnostic(
+    name = 'diag1',
+    grid = grid,
+    period = diagnostic_intervals,
+    data_list = ['rho_electrons', 'rho_he_ions', 'phi'],
+    write_dir = '.',
+    warpx_file_prefix = 'Python_background_mcc_plt'
+)
+
+##########################
+# simulation setup
+##########################
+
+sim = picmi.Simulation(
+    solver = solver,
+    time_step_size = DT,
+    max_steps = max_steps,
+    warpx_collisions=[mcc_electrons, mcc_ions],
+    verbose=True
+)
+
+sim.add_species(
+    electrons,
+    layout = picmi.GriddedLayout(
+        n_macroparticle_per_cell=number_per_cell_each_dim, grid=grid
+    )
+)
+sim.add_species(
+    ions,
+    layout = picmi.GriddedLayout(
+        n_macroparticle_per_cell=number_per_cell_each_dim, grid=grid
+    )
+)
+
+sim.add_diagnostic(field_diag)
+
+##########################
+# simulation run
+##########################
+
+sim.step(max_steps)
+
+##########################
+# print scraped particles
+##########################
+
+n = pywarpx._libwarpx.get_num_particles_impacted_boundary("electrons", 'x_hi')
+print("Number of electrons scraped:", n)
```

Comment:
Instead of just doing this I'd say we should get the weights of the particles in the buffer and save those to a .npy file. Then in the analysis script we should load the yt file gives the particle buffer and confirm that those weight are the same as what you saved in the .npy file.